### PR TITLE
(enhance)(S3) Change s3 metric from bvar adder to latency recorder

### PR DESCRIPTION
**Pull request by ByteYue**
## Proposed changes

Issue Number: close #xxx
We can get better observability like P99, latency etc. with LatencyRecorder.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by ByteYue** [comment]
run buildall

**Comment by ByteYue** [comment]
run buildall

**Comment by ByteYue** [comment]
run buildall

**Comment by ByteYue** [comment]
run buildall

**Comment by ByteYue** [comment]
run buildall

**Comment by ByteYue** [comment]
run buildall

**Comment by ByteYue** [comment]
run buildall

**Comment by ByteYue** [comment]
run buildall

**Comment by ByteYue** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by gavinchou — APPROVED** [review]

**Comment by dataroaring** [comment]
run buildall

**Comment by ByteYue** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by Yukang-Lian — APPROVED** [review]

File: be/src/io/fs/s3_file_system.cpp [hunk 1]
```diff
@@ -425,8 +432,11 @@ Status S3FileSystem::download_impl(const Path& remote_file, const Path& local_fi
     GET_KEY(key, remote_file);
     Aws::S3::Model::GetObjectRequest request;
     request.WithBucket(_s3_conf.bucket).WithKey(key);
-    Aws::S3::Model::GetObjectOutcome response = _client->GetObject(request);
-    s3_bvar::s3_get_total << 1;
+    Aws::S3::Model::GetObjectOutcome response;
+    {
+        SCOPED_BVAR_LATENCY(s3_bvar::s3_get_latency);
```
Comment (SWJTU-ZhangLei) [inline be/src/io/fs/s3_file_system.cpp:437]: the unit is us or ms ? ms better?

**Review by gavinchou — APPROVED** [review]

**Pull request merged**